virt/virt-4-8-release-notes.adoc

```diff
@@ -145,6 +145,47 @@ link:https://access.redhat.com/support/offerings/techpreview[Technology Preview
 [id="virt-4-8-known-issues"]
 == Known issues
 
+* If you initially deployed {VirtProductName} at version 2.4.z or earlier, upgrading to version 4.8 fails with the following message:
++
+[source,terminal]
+----
+risk of data loss updating hyperconvergeds.hco.kubevirt.io: new CRD removes version v1alpha1 that is listed as a stored version on the existing CRD
+----
++
+This bug does not affect clusters where {VirtProductName} was initially deployed at version 2.5.0 or later. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1986989[*BZ#1986989*])
++
+** As a workaround, remove the `v1alpha1` version and resume the upgrade process.
```

Comment:
Change "." to ":" because this is a lead-in to a procedure.

Reply:
we really have to remove `v1alpha1`